contracts/src/core/AlignedLayerServiceManagerStorage.sol

```diff
@@ -1,14 +1,20 @@
+pragma solidity ^0.8.12;
+
 import "eigenlayer-middleware/interfaces/IServiceManager.sol";
 
 contract AlignedLayerServiceManagerStorage {
     struct BatchState {
         uint32 taskCreatedBlock;
         bool responded;
+        address batcherAddress;
     }
 
     /* STORAGE */
     mapping(bytes32 => BatchState) public batchesState;
 
+    // Storage for batchers balances. Used by aggregator to pay for respondToTask
+    mapping(address => uint256) public batchersBalances;
```

Comment:
```suggestion
    mapping(address => uint256) private batchersBalances;
```
there is already a prettier `balanceOf(address) returns (uint256)` in AlignedLayerServiceManager.sol